espnet2/asr/encoder/conformer_encoder.py

```diff
@@ -97,14 +105,17 @@ def __init__(
         macaron_style: bool = False,
         rel_pos_type: str = "legacy",
         pos_enc_layer_type: str = "rel_pos",
-        selfattention_layer_type: str = "rel_selfattn",
+        selfattention_layer_type: str = "legacy_rel_selfattn",
```

Comment:
???
I think it's better not to change it.
Can you tell me why you need to change it?

Reply:
Sorry this might have been a typo- I don't need to change it